pkg/bugzilla/bugzilla.go

```diff
@@ -91,44 +91,26 @@ func (c *Verifier) VerifyBugs(bugs []int) []error {
 				if !prReviewedByQA(comments, reviews, c.pluginConfig.LgtmFor(extPR.org, extPR.repo).ReviewActsAsLgtm) {
 					unapprovedPRs = append(unapprovedPRs, extPR)
 				}
-				labels, err := c.ghClient.GetIssueLabels(extPR.org, extPR.repo, extPR.prNum)
-				if err != nil {
-					newErr := fmt.Errorf("Unable to get labels for github pull %s/%s#%d: %v", extPR.org, extPR.repo, extPR.prNum, err)
-					errs = append(errs, newErr)
-					bugErrs = append(bugErrs, newErr)
-				}
-				var hasLabel bool
-				for _, label := range labels {
-					if label.Name == "qe-approved" {
-						hasLabel = true
-						break
-					}
-				}
-				if !hasLabel {
-					unlabeledPRs = append(unlabeledPRs, extPR)
-				}
 			}
-			bzCFVerified := (len(bug.Verified) == 1 && bug.Verified[0] == "Tested")
-			if len(unapprovedPRs) > 0 || len(unlabeledPRs) > 0 || len(bugErrs) > 0 || !bzCFVerified {
-				message = "Bug will not be automatically moved to VERIFIED for the following reasons:"
+			if len(unapprovedPRs) > 0 || len(bugErrs) > 0 {
+				message = fmt.Sprintf("%s\nBug will not be automatically moved to VERIFIED for the following reasons:", message)
 				for _, extPR := range unapprovedPRs {
-					message = fmt.Sprintf("%s\n- PR %s/%s#%d not appoved by QA contact", message, extPR.org, extPR.repo, extPR.prNum)
-				}
-				for _, extPR := range unlabeledPRs {
-					message = fmt.Sprintf("%s\n- PR %s/%s#%d does not have the qe-approved label", message, extPR.org, extPR.repo, extPR.prNum)
-				}
-				if !bzCFVerified {
-					message = fmt.Sprintf("%s\n- `Verified` field of this bug is not set to `Tested`", message)
+					message = fmt.Sprintf("%s\n- PR %s/%s#%d not approved by QA contact", message, extPR.org, extPR.repo, extPR.prNum)
 				}
 				for _, err := range bugErrs {
 					message = fmt.Sprintf("%s\n- %s", message, err)
 				}
+				message = fmt.Sprintf("%s\n\nThis bug must now be manually moved to VERIFIED", message)
+				// Sometimes the QAContactDetail is nil; if not nil, include name of QA contact in message
+				if bug.QAContactDetail != nil {
+					message = fmt.Sprintf(" by %s", bug.QAContactDetail.Name)
+				}
```

Comment:
These are the lines that changed compared to  the original PR

Reply:
Your truncating your message instead of concatenating it now